src/styles/components/_definition.scss

```diff
@@ -1,12 +1,12 @@
-$breakpoint-background-image-size: 75rem;
+$breakpoint-background-image-size: 75rem; // 1000px/16px
 
 .cmp-definition {
   color: var(--foreground-color);
   width: 100%;
   position: relative;
-  margin: 2.5rem 0; // top/bottom: 40px; left&right: 0;
+  margin: 5rem 0 -5rem; // top/bottom: 40px; left&right: 0;
```

Comment:
Because of the conditional rendering we have for Sandpack and the code examples, Dustin & I decided to get rid of the negative margin here. Otherwise (if we don't render the Sandpack examples), we get something like this: 

<img width="556" alt="Screen Shot 2022-12-16 at 1 22 25 PM" src="https://user-images.githubusercontent.com/69602589/208164097-d33f12f8-9814-4b12-8896-f40081737e61.png">


I love how the negative margin looks when the code examples are in, but Dustin said it was ok to have a little extra space to avoid this, in cases where Sandpack doesn't render.

Reply:
Good call. In cfcb3bc, I made a small change to the HTML to get around this--I gave the sandpack a modifier class that reduces the padding-top. Since the sandpack component is what's conditionally rendering, not the definition, this shouldn't cause issues when conditionally rendering the same way a negative margin on _definition does.

Reply:
Oh great idea! That does exactly what we want! 
<img width="652" alt="Screen Shot 2022-12-16 at 2 50 46 PM" src="https://user-images.githubusercontent.com/69602589/208177907-8d3db6e4-f37e-4ca8-85d4-9d15d54d6dca.png">